dart/test/sentry_client_test.dart

```diff
@@ -613,11 +638,39 @@ SentryEvent? eventProcessorDropEvent(SentryEvent event, {dynamic hint}) {
 }
 
 class Fixture {
+  final MockTransport transport = MockTransport();
+  final transaction = '/test/scope';
+  final eventTransaction = '/event/transaction';
+  final fingerprint = ['foo', 'bar', 'baz'];
+  final eventFingerprint = ['123', '456', '798'];
+  final user = SentryUser(id: '123');
+  final eventUser = SentryUser(id: '987');
+  final crumb = Breadcrumb(message: 'bread');
+  final eventCrumbs = [Breadcrumb(message: 'bread')];
+
+  late SentryEvent event = SentryEvent(
+    level: SentryLevel.warning,
+    transaction: eventTransaction,
+    user: eventUser,
+    fingerprint: eventFingerprint,
+    breadcrumbs: eventCrumbs,
+  );
+
+  late SentryOptions options;
+
   /// Test Fixture for tests with [SentryOptions.sendDefaultPii]
-  SentryClient getSut(bool sendDefaultPii, Transport transport) {
-    var options = SentryOptions(dsn: fakeDsn);
+  SentryClient getSut(bool sendDefaultPii) {
+    options = SentryOptions(dsn: fakeDsn);
     options.sendDefaultPii = sendDefaultPii;
     options.transport = transport;
     return SentryClient(options);
   }
+
+  Scope createScope(SentryOptions options) {
+    return Scope(options)
+      ..user = user
+      ..transaction = transaction
+      ..fingerprint = fingerprint
+      ..addBreadcrumb(crumb);
+  }
```

Comment:
this should be a method within the `sentry_client_test.dart ` file but not really within the Fixture class